src/Classes/ServiceAPI/MyRadio_Selector.php

```diff
@@ -196,11 +196,12 @@ public static function setStudio($studio)
             throw new MyRadioException('Selector Locked');
         } elseif ($response === 'ACK') {
             return [
+            'ready' => self::remoteStreams()['ready'],
```

Comment:
I see what you mean about this, 'ready' isn't exactly the clearest thing in the world for showing what's going on. however, we don't currently have more than one ob audio output so i guess it makes sense...